src/test/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
according to Spring Boot profiling, this file should be named `application-test.properties`

so that you can select the active profile in your tests like:
```
@ActiveProfile("test")
```

read about profiles [here](https://medium.com/javarevisited/getting-started-with-spring-boot-profiles-1e00159f0542)